test/demo/start

```diff
@@ -0,0 +1,40 @@
+#!/usr/bin/env python3
+
+# SPDX-FileCopyrightText: The RamenDR authors
+# SPDX-License-Identifier: Apache-2.0
+
+import sys
+
+from drenv import kubectl
+
+if len(sys.argv) != 2:
+    print(f"Usage: {sys.argv[0]} cluster")
+    sys.exit(1)
```

Comment:
Please match other addons code - in all addons we change the current directory to the
script directory so we can run the addon from any directory.